Project.toml

```diff
@@ -4,12 +4,15 @@ authors = ["Akshay Sharma"]
 version = "0.1.0"
 
 [deps]
+Clp = "e2554f3b-3117-50c0-817c-e040a3ddf72d"
```

Comment:
Move these to `[extras]` and `[targets]` as they are only dependencies for the tests

Reply:
ok. will do it